4.2.5/src/main/java/Towns.java

```diff
@@ -0,0 +1,78 @@
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Towns {
+    public static void main(String[] args) throws IOException {
+        System.out.println("Для начала игры введите название любого города. Для завершения введите слово exit.");
+
+        Scanner input = new Scanner(System.in);
+        String randomTown = input.nextLine();
+
+        List<String> citiesArray = new ArrayList<String>();
+        FileInputStream inputStream = null;
+
+        try {
+            inputStream = new FileInputStream("F:\\Java\\AT-Q4-2019\\_425\\src\\main\\resources\\files\\cities2.txt");
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
```

Comment:
Good! I like it!

Reply:
chenaged on "425/src/main/resources/files/cities2.txt"